examples/chip-tool/commands/pairing/PairingCommand.h

```diff
@@ -42,6 +43,7 @@ enum class PairingNetworkType
     None,
     WiFi,
     Thread,
+    Ethernet,
```

Comment:
Shouldn't this be called `OnNetwork` or something?  The key part is that this network type does not need network credentials; the exact hardware networking technology is not relevant here.

Reply:
Changed to Network to avoid confusion with 'OnNetwork' command